0019-glossary/0019-glossary.md

```diff
@@ -0,0 +1,272 @@
+
+## Liquidity
+Maximum amount which can be moved back and forth over a route.
+
+## Local transaction
+A transaction that contains just one [transfer](#transfer) and zero [hops](#hop), thus staying within the same ledger.
+
+## Outgoing (transfer, ledger, amount)
+Relative to a connector, the transfer/ledger/amount that is located inbetween the connector and the receiver, directly adjacent to the connector.
+For a sender, the adjacent transfer/ledger/amount is always outgoing.
+A receiver does not have an outgoing transfer/ledger/amount.
+
+## Payment
+Roughly synomymous to [transaction](#transaction), but with emphasis on the context of the negotiation between the sender and receiver. For instance,
+a transaction may be paying a debt, or a purchase, it may be a pull payment or a push payment. All these aspects which put a transaction in a wider
+context and link it to real-world interactions, contracts, and motivations between parties make a transaction into a payment. A
+[cyclic transaction](#cyclic-transaction) is generally not considered a payment, because the sending party of such a transaction is also the receiving party.
+
+## Payment channel
+A payment channel is similar to a trustline, but each party parks an asset on an "underlying ledger", equal to the other party's maximum balance.
+Apart from disputes canceling each other out, and out-of-band settlement, disputes can be settled by each party claiming the other party's parked
+asset. This way, two strangers can run a ledger with properties similar to a trustline, but without fully trusting each other.
+
+## Peer balance
+See [trustline](#trustline).
+
+## Peering
+Connectors "peer" with one another to exchange information used to determine the best route for a payment, and to route it (connect its chain of
+transfers through [forward](#forward-action) and [backward](#backward-action) action.
+A peering relationship is usually implemented using either a [trustline](#trustline) or a [payment channel](#payment-channel)..
+
+## Prepared
+The second state of an interledger transaction, after [proposed](#proposed), and
+before its [final state](#final-state).
+
+## Pre-Shared Key (PSK)
+The recommended transport layer for most use cases.
+In Pre-Shared Key (PSK) protocol, the sender and receiver use a shared secret to generate the payment condition, authenticate the ILP packet, and encrypt application data. Using PSK, the sender is guaranteed that fulfillment of their transfer indicates the receiver got the payment, provided that no one aside from the sender and receiver have the secret and the sender did not submit the fulfillment. See [IL-RFC 16](../0016-pre-shared-key/0016-pre-shared-key.md).
+
+## Proposed
+The initial state of an interledger transaction, before [prepared](#prepared).
+
+## Receiver
+The party who is the beneficiary (payee) of a transfer or a transaction (payment).
+
+## Route
+A list of [transfers](transfer) and the [hops](#hop) between them.
```

Comment:
This can refer to both the "route" an actual payment has taken or the theoretical way that a future payment could go. The former is sometimes also called the path